eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/VariantContextWriterTest.java

```diff
@@ -350,8 +343,7 @@ private void assertFlagIsPresent(String flagRegex) throws Exception {
                        buildVariant(CHR_1, 1000, "C", "G", SNP_SEQUENCE_ONTOLOGY, true, true, false, false, false,
                                     STUDY_1));
 
-        String dataLinesRegex = "^[^#]";
-        String dataLinesWithValidatedRegex = dataLinesRegex + somewhereSurroundedByTabOrSemicolon(flagRegex);
+        String dataLinesWithValidatedRegex = DATA_LINES_REGEX + somewhereSurroundedByTabOrSemicolon(flagRegex);
```

Comment:
Are the substitutions equivalent? DATA_LINES_REGEX has an extra .* right?

Reply:
Agreed, this may be fine when initializing `dataLinesWithFlagsRegex` but not the rest.

Reply:
`somewhereSurroundedByTabOrSemicolon` is always used in combination of DATA_LINES_REGEX, so I did a bit of refactoring